orchestration/dags/import_analytics.py

```diff
@@ -286,6 +286,26 @@ def getting_service_account_token():
     dag=dag,
 )
 
+create_offer_extracted_data = BigQueryOperator(
+    task_id="create_offer_extracted_data",
+    sql=f"""Select B.offer_id, B.offer_type, json_extract(offer_extra_data, "$.author") AS Author, 
+            json_extract(offer_extra_data, "$.performer") AS Performer, 
+            json_extract(offer_extra_data, "$.MusicType") AS MusicType, 
+            json_extract(offer_extra_data, "$.MusicSubtype") AS MusicSubtype,
+            json_extract(offer_extra_data, "$.stageDirector") AS stageDirector, 
+            json_extract(offer_extra_data, "$.theater") AS Theater,
+            json_extract(offer_extra_data, "$.showType") AS showType,
+            json_extract(offer_extra_data, "$.showSubType") AS showSubType,
+            json_extract(offer_extra_data, "$.speaker") AS speaker,
+            json_extract(offer_extra_data, "$.rayon") AS rayon
+            from (SELECT offer_extra_data, offer_id, offer_type
+                    FROM `{GCP_PROJECT}.{BIGQUERY_ANALYTICS_DATASET}.applicative_database_offer`) B""",
```

Comment:
Pourquoi ne pas mettre un FROM classique ?
```suggestion
          FROM `{GCP_PROJECT}.{BIGQUERY_ANALYTICS_DATASET}.applicative_database_offer`""",
```